app/src/main/java/com/simformsolutions/oneloop/ui/user/signup/signupuserinfo/SignupUserInfoScreen.kt

```diff
@@ -0,0 +1,331 @@
+    onLastNameChange: (String) -> Unit,
+    onUserInfoNextClicked: () -> Unit,
+    pickImage: () -> Unit,
+    onSignInClick: () -> Unit
+) {
+    AppScaffold(
+        modifier = modifier,
+        topBar = {
+            AppTopAppBar(
+                containerColor = AppTheme.appColorScheme.blackColor,
+                centerContent = {
+                    AppText(
+                        text = stringResource(R.string.sign_up),
+                        textColor = AppTheme.appColorScheme.white
+                    )
+                },
+                leadingContent = {
+                    Row(
+                        modifier = Modifier
+                            .clickable { onBackClick() }
+                            .padding(
+                                start = dimensionResource(R.dimen.common_space_small),
+                                end = dimensionResource(R.dimen.common_space_very_small)
+                            ),
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        AppIcon(
+                            modifier = Modifier
+                                .padding(end = dimensionResource(R.dimen.common_space_very_small)),
+                            painter = painterResource(id = R.drawable.icon_back_arrow)
+                        )
+
+                        AppText(
+                            text = stringResource(R.string.back),
+                            textColor = AppTheme.appColorScheme.white
+                        )
+                    }
+                }
+            )
+        }
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .verticalScroll(rememberScrollState())
+                .paint(
+                    painter = painterResource(id = R.drawable.img_login_bg),
+                    contentScale = ContentScale.FillBounds
+                )
+                .imePadding()
+                .padding(innerPadding)
+                .padding(
+                    start = dimensionResource(id = R.dimen.main_contains_side_space),
+                    end = dimensionResource(id = R.dimen.main_contains_side_space)
+                ),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center
+        ) {
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(
+                        bottom = dimensionResource(id = R.dimen.app_icons_bottom_space)
+                    ),
+            ) {
+                AppIcon(
+                    modifier = Modifier
+                        .align(Alignment.TopCenter)
+                        .padding(dimensionResource(id = R.dimen.common_space_large)),
+                    painter = painterResource(id = R.drawable.icon_avatar_border)
+                )
+
+                Box(
```

Comment:
We might should be able to achieve without using the inner box. Please give it a try.